src/sestest/readme.md

```diff
@@ -74,6 +75,7 @@ The `generate` mode allows you to generate a software entitlement token with the
 | --not-after   | Optional  | The moment at which the token expires and the application is no longer entitled to execute <br/> Format: 'yyyy-mm-ddThh-mm'; 24 hour clock; local time; defaults to 7 days from now. |
 | --address     | Optional  | The IP addresses of the machine entitled to execute the application(s). <br/> Defaults to all the IP addresses of the current machine.                                               |
 | --audience    | Optional  | Audience to the token will be addressed. <br/> Defaults to `https://batch.azure.test/software-entitlement`.                                                                          |
+| --issuer      | Optional  | Issuer by whom the tokens are created. <br/> Defaults to `http://batch.azure.test/software-entitlement`.                                                                             |
```

Comment:
Elsewhere you say 'which' rather than 'whom'. (I know, I am really focusing on the issues today.)